react/week4/searcher/src/SearchResult.js

```diff
@@ -0,0 +1,37 @@
+import {useContext} from 'react';
+import {SearchContext} from './SearchContext';
+
+const SearchResult = () => {
+    const searchContext = useContext(SearchContext);
+
+    return (
+        <div>
+            {searchContext.isLoading
+            ? 'Loading...'
+            : searchContext.error.length > 0
+                ? <p>{searchContext.error}</p>
+                : <UserList />
+            }
```

Comment:
It's hard to read the code when there's nested ternaries like here. 
I'd move the logic up, like:
```
const searchResult  = searchContext.error.length > 0 ? ...

return (
  <div> ...
)
```